.github/workflows/amd-aie-distro.yml

```diff
@@ -155,11 +155,11 @@ jobs:
           - OS: ubuntu-20.04
             ARCH: x86_64
 
-          - OS: ubuntu-20.04
-            ARCH: aarch64
+          # - OS: ubuntu-20.04
+          #   ARCH: aarch64
 
-          - OS: windows-2019
-            ARCH: AMD64
+          # - OS: windows-2019
+          #   ARCH: AMD64
```

Comment:
Windows doesn't work because 

```
CMake Error at runtimes/CMakeLists.txt:439 (message):
  libc-hdrgen executable missing
```

Didn't investigate...